zebra/zebra_rib.c

```diff
@@ -955,16 +937,13 @@ static unsigned nexthop_active_check(struct route_node *rn,
  * Return value is the new number of active nexthops.
  */
 
```

Comment:
Similarly, comment refers to `set` argument

Reply:
fixed